dev/podman-kind.md

```diff
@@ -14,10 +14,15 @@ brew install podman
 ```
 Once done, init and start the machine:
 ```
-podman machine init
+podman machine init --volume $HOME:$HOME
 podman machine start
 ```
 
+## Install system helper service
+```
+sudo /usr/local/Cellar/podman/4.0.3/bin/podman-mac-helper install
```

Comment:
We should note that this is only a one-time thing

Reply:
Also, the user needs to run the other `export` command to set `$DOCKER_HOST` correct?

Reply:
The command is an alternative to DOCKER_HOST export.

Reply:
Ahh I didn't see that... sounds good